_posts/2018-01-19-choose-reviewers.md

```diff
@@ -0,0 +1,110 @@
+Arcas allows me to collect articles' meta data by specifying just a single keyword.
+
+Using the following few lines of code I am able to collect up to 100 articles
+that my reviewers have written and published on the journal [Springer](http://www.springer.com/gb/).
+Arcas allows you to ping 5 different sources but for the purpose of this work a
+single source, more specifically Springer works just fine.
+
+{% highlight python %}
+>>> import pandas as pd
+>>> import requests
+>>> import arcas
+>>> for p in [arcas.Springer]:
+...   for author in names:
+...       api = p()
+...       switch = True
+...       start = 1
+...       while switch is not False and start < 100:
+...           parameters = api.parameters_fix(author=author, records=10, start=start)
+...           url = api.create_url_search(parameters)
+...           request = api.make_request(url)
+...           root = api.get_root(request)
+...           raw_article = api.parse(root)
+...           try:
+...               articles = []
+...               for art in raw_article:
+...                   articles.append(api.to_dataframe(art))
+...               df = pd.concat(articles, ignore_index=True)
+...               api.export(df, "{}_{}.json".format(author, start))
+...           except TypeError:
+...               switch=False
+...           start += 10
+{% endhighlight %}
+
+Once the data have been collected I need to narrow down the reviewers that can
+understand the science? of my work or its’ applications. For my third task
+I used the library [pandas](https://pandas.pydata.org/)
+to check whether the keywords of my paper exist within any of the abstracts of
+my reviewers' articles.
+
+{% highlight python %}
+>>> keywords = ['keyword one', 'keyword two', 'keyword three']
+>>> def find_reviewers(names, keywords):
+...    authors_with_no_articles = []
+...    authors_of_interest = []
+...    for name in names:
+...        try:
+...            df = read_author_files(name)
+...            df = df.dropna()
+...            for key in keywords:
+...                flag = check_keyword_in_abstract(key, df)
+...                if flag == True:
+...                    authors_of_interest.append((name, key))
+...        except ValueError:
+...            authors_with_no_articles.append(name)
+...    return authors_of_interest, authors_with_no_articles
+>>> keep, no_keep = find_reviewers(names, keywords)
+{% endhighlight %}
+
+Finally, I can  count the number of words each reviewer satisfied and plot a histogram!
+Thus the potential reviewers of my paper are the people who rank first on the following
+histogram.
+
+<p align="center">
+ <img src="{{site.baseurl}}/assets/images/reviewers_hist.png" style='height: 60%; width: 80%; object-fit: contain'>
+</p>
+
+A manual check of the reviewers that stand out must be done as well. I am not
+implying that one should follow the histogram by heart but it was a fast and
```

Comment:
`by heart` means doing something by memory so it's not quite right here. Perhaps `strictly follow the histogram`.